### DISCUSSION TITLE
Is keycloak affected by CVE-2023-46604

### DISCUSSION THREAD

**Discussion by khnn** [keycloak/keycloak#24573] [Q&A] [answered]
Hey everyone, Is Keycloak affected by https://cve.mitre.org/cgi-bin/cvename.cgi?name=2023-46604?

**Comment by abstractj** [comment]
@khnn the CVE-2023-46604 refers to a critical vulnerability in Apache ActiveMQ. Could you please elaborate more about how this CVE would be related with Keycloak?

**Comment by mvk37** [comment, marked as answer]
Quarkus based Keycloak is not including ActiveMQ and is not vulnerable. 

WildFly includes ActiveMQ Artemis as a dependency. But according to this discussion https://lists.apache.org/thread/hp07tp8k9m15z2jzzjh4wt5l1vcj273b it doesn't include affected ActiveMQ jars. So it also is not vulnerable.
